UIforETW/UIforETWDlg.cpp

```diff
@@ -295,7 +294,7 @@ BOOL CUIforETWDlg::OnInitDialog()
 	}
 
 	wchar_t* windowsDir = nullptr;
-	VERIFY(SUCCEEDED(SHGetKnownFolderPath(FOLDERID_Windows, 0, NULL, &windowsDir)));
+	ATLVERIFY(SUCCEEDED(SHGetKnownFolderPath(FOLDERID_Windows, 0, NULL, &windowsDir)));

```

Comment:
If we made this a UIETWVERIFY we could resolve the /analyze warnings on these functions. Later.